src/test/java/no/unit/nva/handlers/AddRoleHandlerTest.java

```diff
@@ -0,0 +1,137 @@
+package no.unit.nva.handlers;
+
+import static nva.commons.utils.JsonUtils.objectMapper;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.mockito.Mockito.mock;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Optional;
+import no.unit.nva.database.DatabaseService;
+import no.unit.nva.database.DatabaseServiceImpl;
+import no.unit.nva.database.DatabaseTest;
+import no.unit.nva.database.exceptions.DataHandlingError;
+import no.unit.nva.database.exceptions.InvalidInputRoleException;
+import no.unit.nva.database.exceptions.InvalidRoleException;
+import no.unit.nva.database.exceptions.UnexpectedException;
+import no.unit.nva.model.RoleDto;
+import no.unit.nva.testutils.HandlerRequestBuilder;
+import nva.commons.handlers.GatewayResponse;
+import org.apache.http.HttpStatus;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.zalando.problem.Problem;
+
+public class AddRoleHandlerTest extends DatabaseTest {
+
+    public static final String SOME_ROLE_NAME = "someRoleName";
+    private AddRoleHandler addRoleHandler;
+    private Context context;
+
+    /**
+     * init.
+     */
+    @BeforeEach
+    public void init() {
+        initializeTestDatabase();
+        context = mock(Context.class);
+        DatabaseService service = new DatabaseServiceImpl(localDynamo);
+        addRoleHandler = new AddRoleHandler(mockEnvironment(), service);
+    }
+
+    @Test
+    public void addRoleHandlerHasProcessInputMethod()
+        throws DataHandlingError, UnexpectedException, InvalidRoleException, InvalidInputRoleException {
+        RoleDto roleDto = RoleDto.newBuilder().withName(SOME_ROLE_NAME).build();
+        addRoleHandler.processInput(roleDto, null, null);
```

Comment:
Not sure what you test here…not throwing MethodNotFoundException?

Reply:
This test would throw a compilation error if it failed.